app.html

```diff
@@ -1114,6 +1130,75 @@
   }
 }    
 
+var rawHidFilter = { vendorId: 0x16C0, productId: 0x0486, usagePage: 0xFFAB, usage: 0x0200 }; // RawHID interface
+var rawHidParams = { filters: [rawHidFilter] };
+var hid_device;
+var outputReportId = 0x00;
```

Comment:
variable should have "hid" in it somewhere

Reply:
Done

Reply:
done